table.csv

```diff
@@ -21,8 +21,9 @@ keccak-512,                     multihash,      0x1d,           draft,
 blake3,                         multihash,      0x1e,           draft,     BLAKE3 has a default 32 byte output length. The maximum length is (2^64)-1 bytes.
 sha2-384,                       multihash,      0x20,           permanent, aka SHA-384; as specified by FIPS 180-4.
 dccp,                           multiaddr,      0x21,           draft,
-murmur3-128,                    multihash,      0x22,           draft,
+murmur3-64,                     multihash,      0x22,           draft,
 murmur3-32,                     multihash,      0x23,           draft,
+murmur3-128,                    multihash,      0x24,           draft,
```

Comment:
Do people use this? Should it be higher up in the table? I'm willing to wait on a user here asking for this before allocating a codec.

Also, murmur3-128 is not a defined thing. There is a 32bit and 64bit version of it. So we'd need a codec for each of the ones we care about. (e.g. murmur3-128-x86 and murmur3-128-x64)

Reply:
I think you need to look at those as `0x22` which has "murmur3-128" here already. Let's just fix up `0x22` to have a better name, or fix up the uses of it such that it's clear what kind of sorcery is being invoked and not add new entries. Nobody is happy with murmur3 in this table anymore so lets not expand it, at least here in the 1 byte range.